app/models/content_page.rb

```diff
@@ -1,9 +1,22 @@
 class ContentPage < ApplicationRecord
   acts_as_tree
+
   scope :top_level, -> { where("parent_id IS NULL") }
+  scope :order_by_position, -> { order("position ASC") }
 
-  validates :title, presence: true, uniqueness: true
+  validates :title, format: { with: /\A[a-zA-Z0-9 ]+\Z/, message: "Title should only contain alphabet, numeric and space characters" }, presence: true, uniqueness: true
```

Comment:
need to remove the word title from this string to make it consistent with the other messages

![Uploading Screenshot 2021-02-05 at 15.14.58.png…]()

Reply:
Yep, I need to refactor all mentioned of title and subtitle :-(